test/integration_new/__init__.py

```diff
@@ -1,18 +1,26 @@
-import configparser
+import base64
 import os
-from pathlib import Path
+import json
 
 from boxsdk.auth.jwt_auth import JWTAuth
 from boxsdk.client import Client
 
 
-def read_jwt_path_from_config(config_path: str):
```

Comment:
Have you considered supporting .json configuration when running integration tests locally? It may be easier for some developers to download a configuration file and copy it into this particular solution solution, rather than setting an env variable encoded as base64 for the entire machine. For example, our .NET SDK looks for the env variable and, if it is not present, it takes the .json configuration found in the base dir.